packages/main/src/themes/PopupsCommon.css

```diff
@@ -55,6 +55,7 @@
 
 .ui5-popup-content {
     overflow: auto;
+    scrollbar-gutter: stable;
```

Comment:
Is it supported on Safari
https://developer.mozilla.org/en-US/docs/Web/CSS/scrollbar-gutter

There is some related issue, we need to look at it
https://bugs.webkit.org/show_bug.cgi?id=167335

Reply:
You are right. The property isn't currently supported for Safari.